### PR TITLE
fix should should test typo

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
Just fixing a simple typo I came across 🙂

### CODE REVIEW

**Review by GregBrimble — APPROVED** [review]

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Easy win! Nice thanks @dario-piotrowicz.
Let's wait till we have CI up and running again before we land this.

**Comment by JacobMGEvans** [comment]
Rebased on the reverted Test fix #2648

**Pull request closed without merging**
